CHANGELOG

```diff
@@ -1,5 +1,170 @@
+* Fix #7493: Remove jinja from base.py (#9294)
+* Milestone 1.3.2: Import WaveSurfer using yarn and add typedefs for it (#9279)
+* Fix part of #8833: Makes all event listeners passive by default (#9178)
+
+Data handling
+* Fix part of #5914: Complete Takeout Frontend (#9424)
+* Fix part of #5914: Add JSON rendering testing to Takeout (#9301)
+* Fixed user updates. (#9252)
+*  Added update username functionality in the admin tab. (#9191)
+* Wipeout 3.10: Add new field to rights models (#8505)
+
+QA Team
+* Fix #9442: Delete Un-Migrated E2E Test Code (#9444)
+* Fix #9372: Mocked datetime to fix admin test flakyness. (#9394)
+* Fix part of #4057: Tests for editable-exploration-backend-api.service.ts, RecordedVoiceoversObjectFactory.ts,  and topic-update.service.ts (#9380)
+* Fix #8924: Separate inline AngularJS modal controllers into new files (2nd) (#9376)
+* Extend ExplorationMissingStatsAudit to check for state stats (#9361)
+* Fix #8924 Separate inline AngularJS modal controllers into new files (1st) (#9344)
+* Introduce ExplorationMissingStatsAudit One Off Job (#9306)
+* Fix #9132: Transition E2E Tests to Async-Await (#9267)
+* Fix #8758: add waitFor statement in getResponseEditor method (#9229)
+* Increase coverage for autosave-info-modals.service.ts (#9174)
+* Fix part of #8924: Create a generic modal and replace it and inherit it in common and similar inline modals from $uibModal.open controller property (#9167)
+* Fix part of #4057: Added a test file for exploration-param-specs.service.ts (#9115)
+
+Angular Migration
+* Adds all Angular services to UpgradedServices (#9416)
+* Add i18n translation service for angular code! (#9398)
+* Fix part of #8015: return CollectionRights rather than dict (#9389)
+* Fix part of #7165: Add WARNING_TYPE to interaction validation spec files (#9371)
+* Fix part of #7176: Remove any from some object factories - III (#9370)
+* Fix part of #7176: Remove any from RuleObjectFactory and introduce TypeChangeService (#9365)
+* Fix part of #7176: Remove any types from some object factories - II (#9349)
+* Fix part of #8472: Upgrades CurrentInteractionService to Angular (#9347)
+* Fix part of #7176: Add type definitions in some object factories (#9345)
+* Fix part of #8472: Upgrades ExplorationRecommendationsService to Angular (#9339)
+* Fix part of #8472: Upgrades UserExplorationPermissionsService to Angular (#9338)
+* Migrates Background Banner directive to Angular Component (#9330)
+* Fix part of #8472: Upgrade StateInteractionStatsService to Angular 8 (#9321)
+* Change controllers to components (Part 1) (#9317)
+* Fix part of #8742: Upgrade AnswerClassificationService to Angular 8 (#9313)
+* Milestone 1.4.3: Add type definitions for __fixtures__ (#9307)
+* Fix #8472: Collection creation service upgrade (#9303)
+* Fix part of #8472: Upgrades StatsReportingService and MessengerService to Angular (#9299)
+*  Milestone 1.4.2: Split ICustomScope to respective directives (#9298)
```

Comment:
Ditto

Reply:
Fixing directly on release branch.

Reply:
note: you probably also want to do a fix to develop (later is fine) to maintain parity

Reply:
Done here: https://github.com/oppia/oppia/pull/9724